.github/workflows/docker_images_template.yaml

```diff
@@ -25,12 +25,12 @@ jobs:
 
       - name: Build image
         run: |
+          PYPI_TAG=${{steps.get-ref.outputs.tag}}
           curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/${{inputs.wmcore_component}}/Dockerfile
```

Comment:
With the separation of dockerfiles, this is no longer an issue. It is also not clear to me what the `svn` dependency would incur here, so I'd rather leave it for a future development, in case we deem it necessary.

Reply:
hi @amaltaro 
> With the separation of dockerfiles, this is no longer an issue.

This is not exactly correct. In order to separate the images you had to move things/files around, which is about to cause other issues at runtime. While the separation of images is a good thing as I said multiple times, this is in no way a substitute of a must fix here. And such separation should definitely be done without the need of shuffling an already established construction of the original image, so that this separation does not affect steps further down the line and causing effects most of the times revealing themselves at runtime but not at build time.   

> It is also not clear to me what the svn dependency would incur here

And here is the full list of dependencies for `svn`: 
```
apt-cache show subversion
Package: subversion
Version: 1.10.4-1+deb10u3
Installed-Size: 4809
Maintainer: James McCoy <jamessan@debian.org>
Architecture: amd64
Depends: libsvn1 (= 1.10.4-1+deb10u3), libapr1 (>= 1.5.0), libaprutil1 (>= 1.3.2+dfsg), libc6 (>= 2.4), libsasl2-2 (>= 2.1.27+dfsg)
```

Merrily 5 libraries. And on a standard Debian based system, the full set of dependencies (excluding `libc6` and `libsasl2`, which must have been satisfied already due to their multiple forward dependencies), the added size is exactly 9946 kB. See bellow:
```
apt-get install subversion
...
The following additional packages will be installed:
  libapr1 libaprutil1 libserf-1-1 libsvn1
...
0 upgraded, 5 newly installed, 0 to remove and 322 not upgraded.
Need to get 0 B/2,658 kB of archives.
After this operation, 9,946 kB of additional disk space will be used.
```   

I do not see a reason why we should make our life harder by striping the capabilities of the whole containerization system only because we decided not to sync the whole needed project area in our CI/CD pipeline. This implies restrictions on our work. Just because the capabilities of the CI/CD system differ from the manual builds and the standard way of how Docker is used, we doom ourselves to indefinite remembering of an obstacle, forcing us to create multiple future workarounds. We can see how this already stands in the way of the `WMAgent`  image creation. The proper way of fixing the issue is not every time we stumble on it to create a workaround in the component's deployment which suffers from the obstacle we have nailed here, but rather fixing the obstacle itself once and forever. Moreover, provided  the cost for such a fix is less than 10 MB.

Reply:
No, that is not correct! I simply move files from the `wmagent` directory over to the `wmagent-base` one, keeping those files added under the same directory. So I fail to understand why you say it will not work and why there is yet a fix that must be done.

In addition, it is not because we started with those scripts under wmagent dir that we need to end with them in there. Keep in mind that this was just an initial work and there is still many developments before we get to something more palpable.

Reply:
> No, that is not correct! I simply move files from the wmagent directory over to the wmagent-base one, keeping those files added under the same directory

No this is not correct. You do change destination of files. And this even though may go well at build time will break at runtime. Please take a look at my other PR.